app/models/concerns/id.rb

```diff
@@ -1,11 +1,12 @@
 # frozen_string_literal: true
 
-module Paginable
```

Comment:
Because `recent` is not always used with pagination. `ID` is a concern for any models with incremental IDs.
I'm considering a better naming...